src/main/java/com/infamous/dungeons_animations/DAEvent.java

```diff
@@ -0,0 +1,48 @@
+package com.infamous.dungeons_animations;
+
+import com.infamous.dungeons_animations.goals.AvoidAndApproachTargetGoal;
+import com.infamous.dungeons_animations.goals.Evoker.SpellAttackGoal;
+import com.infamous.dungeons_animations.goals.Evoker.SummonVexGoal;
+import com.infamous.dungeons_animations.goals.GeoVindicatorAttackGoal;
+import com.infamous.dungeons_animations.goals.GeoVindicatorMeleeGoal;
+import com.infamous.dungeons_animations.mod.DAEntityType;
+import net.minecraft.world.entity.ai.attributes.Attributes;
+import net.minecraft.world.entity.ai.goal.GoalSelector;
+import net.minecraft.world.entity.ai.goal.MeleeAttackGoal;
+import net.minecraft.world.entity.monster.Evoker;
+import net.minecraft.world.entity.monster.Vindicator;
+import net.minecraftforge.event.entity.EntityAttributeCreationEvent;
+import net.minecraftforge.event.entity.EntityJoinLevelEvent;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import static com.infamous.dungeons_animations.DungeonsAnimations.MODID;
+
+@Mod.EventBusSubscriber(modid = MODID)
+public class DAEvent {
```

Comment:
I am going to assume DA stands for Dungeons Animations. 

You should never do "big ass events classes" I know this is fairly standard practice in the MC modding community, but it isn't good for a team.